### PR TITLE
Updated build target

### PR DESCRIPTION
**Pull request by sarahscott**
Updated build target so tests can run.

### CODE REVIEW

File: Example/Artsy+OSSUIFonts.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -492,7 +492,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.1;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
```
Comment (alloy) [inline Example/Artsy+OSSUIFonts.xcodeproj/project.pbxproj]: This should probably be 10.0 ?

File: Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json [hunk 2]
```diff
@@ -18,9 +18,10 @@
   },
   "source_files": "Pod/Classes",
   "resources": "Pod/Assets/*",
+  "preserve_paths": "Pod/Scripts/*",
   "frameworks": [
     "UIKit",
     "CoreText"
   ],
-  "prepare_command": "echo \"HELLO\"\necho $PWD\necho $ENV"
+  "prepare_command": "./Pod/Scripts/ArtsySetup.rb"
```
Comment (alloy) [inline Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json:26]: Was this previous command important for anything?

**Comment by alloy** [comment]
Should also update the target in the podspec https://github.com/artsy/Artsy-OSSUIFonts/blob/master/Artsy%2BUIFonts.podspec#L11-L12

**Comment by alloy** [comment]
@orta Did we already decide to drop iOS 9 support?

**Comment by orta** [comment]
I don't really see a reason to force this lib to be 10+, it doesn't use any APIs and we have it in other projects that still support 9

**Comment by alloy** [comment]
Agreed. @sarahscott _Why_ did the tests not run before?

**Comment by sarahscott** [comment]
@alloy the lowest possible deployment target that I can see in my version of Xcode is 8.0, so I couldn't run tests locally. If we need to leave it at 7, we can do that and it shouldn't be a problem for Emission. Otherwise, I can change it to 8.0 or 9.0.

**Comment by alloy** [comment]
Ah I see, yeah then do 8.

File: Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json (hunk 2 above)
Comment (sarahscott) [reply Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json:26]: I didn't change this myself so I'm not totally sure why it changed or what the old one is for.

Comment (sarahscott) [reply Example/Pods/Local Podspecs/Artsy+UIFonts.podspec.json:26]: Oh @alloy - this podspec was old. When I `pod update`d, it changed to match the current one on `master`.

**Comment by sarahscott** [comment]
Closing in favor of #14

**Pull request closed without merging**